packages/libs/components/src/map/MapVEuMap.tsx

```diff
@@ -509,19 +495,15 @@ function CustomZoomControl(props: CustomZoomControlProps) {
           role="button"
           aria-label="Zoom out"
           onClick={zoomOut}
+          style={{
+            color:
+              map.getZoom() === props.defaultViewport?.zoom
```

Comment:
Sorry, a couple of other things came up.

Just greying it out is OK but not ideal.

If the default zoom level was 3, as below, then the button would still function and zoom out to level 2. 

```
export const defaultAppState: AppState = {
  viewport: {
    center: [0, 0],
    zoom: 3,
  },
```

Reply:
So I think better to disable the button functionally as well as aesthetically.

Also, we should have the same logic for the zoom **in** button.

I don't think the defaultAppState zoom level should be used for either limit.  We should use the Leaflet `map` object's minzoom and maxzoom properties (assuming these are readable).

Reply:
Finally, these Leaflet buttons are "leaky". You can drag or double click on them and it is the same as dragging/double clicking elsewhere in the map. Could also cause problems if there are markers under the buttons.  Could you prevent click events passing through please? (I don't know how, but I note that our custom floaters, like the legend, behave OK.)

Reply:
@bobular Thank you for your comments and suggestions 👍 I will think about the first two. For the last, actually it is leaflet's default behavior. That is, even the default leaflet's zoom control has the same functionality: you can test it at https://leafletjs.com/examples/quick-start/. If you still want it, I will think about how to override it.

Reply:
@bobular I have addressed all of your feedbacks and they seem to work fine. Can you please check them out? Thanks!